File: api/src/Entity/Tag.php
<?php

namespace App\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\HttpFoundation\Tests\StringableObject;
use App\Entity\Base\BaseEntity;


/**
 * @ORM\Entity()
 * @ORM\Table(name="tags")
 */
class Tag extends BaseEntity {

    public function __construct() {
        parent::__construct();
        $this->tag_documents = new ArrayCollection();
    }

    /**
     * @ORM\Id
     * @ORM\Column(type="string")
     * @ORM\GeneratedValue(strategy="UUID")
     */
    protected $tag_id;

    /**
     * @ORM\Column(type="string")
     */
    protected $tag_name;

    public function getTagId(){
        return $this->tag_id;
    }

    public function setTagName($name){
        $this->tag_name = $name;
        return $this;
    }

    public function getTagName(){
        return $this->tag_name;
    }

    /**
     * @ORM\ManyToMany(targetEntity="Document", mappedBy="document_tags")
     * @var Documents[]
     */
    protected $tag_documents;
}

File: openex-player/src/main/java/io/openex/database/model/Tag.java
package io.openex.database.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import org.hibernate.annotations.GenericGenerator;

import javax.persistence.*;

@Entity
@Table(name = "tags")
public class Tag implements Base {
    @Id
    @Column(name = "tag_id")
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    @JsonProperty("tag_id")
    private String id;

    @Column(name = "tag_name")
    @JsonProperty("tag_name")
    private String name;

    @Override
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
